bigbluebutton-html5/imports/ui/components/app/component.jsx

```diff
@@ -368,14 +409,8 @@ class App extends Component {
         <div className={styles.slide}>
           <Button
             hideLabel
-            onClick={
-             () => {
-               (chatWidth !== CHAT_MAX_WIDTH)
-                 ? this.setState({ chatWidth: CHAT_MAX_WIDTH, toggleChatLayout: true })
-                 : this.setState({ chatWidth: CHAT_MIN_WIDTH, toggleChatLayout: false });
-             }
-            }
-            size="sm"
+            onClick={() => this.toggleChatPanel()}
```

Comment:
In the constructor of this component you need to add the below to ensure the relevance of "this" is set to component. Javascript is tricky in these cases

this.toggleChatPanel= this.toggleChatPanel.bind(this);